subsys/bluetooth/host/gatt.c

```diff
@@ -2542,20 +2593,143 @@ int bt_gatt_notify_cb(struct bt_conn *conn,
 int bt_gatt_notify_multiple(struct bt_conn *conn, uint16_t num_params,
 			    struct bt_gatt_notify_params *params)
 {
-	int i, ret;
+	int i;
+	int err;
+	size_t total_len = 0;
+	struct net_buf *buf;
+	struct notify_data data;
 
 	__ASSERT(params, "invalid parameters\n");
 	__ASSERT(num_params, "invalid parameters\n");
 	__ASSERT(params->attr, "invalid parameters\n");
 
+	/* Send any outstanding notifications */
+	gatt_notify_flush(conn);
+
+	CHECKIF(num_params == 1) {
```

Comment:
Suggest to remove `__ASSERT(num_params, "invalid parameters\n");`and add the `0` check here as well